js/ui/windowManager.js

```diff
@@ -140,28 +142,144 @@ var DisplayChangeDialog = new Lang.Class({
     },
 });
 
+// ControllableShellWobblyEffect
+//
+// "Metaclass" that exists to store settings for an effect
+// to be attached to a surface. It has a few properties
+// which are modified by the AnimationsDbus.ServerEffect that
+// owns it when ChangeSetting is called on its owner object. It
+// also has a createActorPrivate() method which creates an
+// EOSShellWobbly, representing private data for an attached
+// effect to an actor.
+const ControllableShellWobblyEffect = new Lang.Class({
+    Name: 'ControllableShellWobblyEffect',
+    Implements: [ AnimationsDbus.ServerEffectBridge ],
+    Extends: GObject.Object,
+    Properties: {
+        spring_k: GObject.ParamSpec.double('spring-k',
+                                           'Spring K',
+                                           'The Spring Constant to use',
+                                           GObject.ParamFlags.READWRITE |
+                                           GObject.ParamFlags.CONSTRUCT,
+                                           2.0,
+                                           10.0,
+                                           8.0),
+        friction: GObject.ParamSpec.double('friction',
+                                           'Friction',
+                                           'The Friction Constant to use',
+                                           GObject.ParamFlags.READWRITE |
+                                           GObject.ParamFlags.CONSTRUCT,
+                                           3.0,
+                                           10.0,
+                                           5.0),
+        slowdown_factor: GObject.ParamSpec.double('slowdown-factor',
+                                                  'Slowdown Factor',
+                                                  'How much to slow animations down',
+                                                  GObject.ParamFlags.READWRITE |
+                                                  GObject.ParamFlags.CONSTRUCT,
+                                                  1.0,
+                                                  5.0,
+                                                  1.0),
+        object_movement_range: GObject.ParamSpec.double('object-movement-range',
+                                                        'Object Movement Range',
+                                                        'How far apart control points can be from each other',
+                                                        GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
+                                                        10.0,
+                                                        500.0,
+                                                        100.0)
+    },
+
+    _init: function(params) {
+        this.parent(params);
+    },
```

Comment:
Remove this, it's the default implementation

Reply:
Fixed